.github/workflows/tag-release.yml

```diff
@@ -9,6 +9,7 @@ on:
   push:
     branches:
       - main
+      - v2
```

Comment:
Will we still need a v2 branch after our release discussion?
Thought we will keep merging all milestone 2 features to main and then release main's changes to Packagist as `2.0.0-preview`?

Please correct me.

Reply:
Or consider a regex pattern to cover future major release branches (https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags)

Reply:
Yes that's correct.

Reply:
Maybe this could be used for future breaking changes after the coming breaking change.

Reply:
Might be useful to have a dynamic filter for the major upgrades... https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags